.mvn/javadoc/package-list

```diff
@@ -10,6 +10,7 @@ nl.talsmasoftware.context.futures
 nl.talsmasoftware.context.metrics
 nl.talsmasoftware.context.locale
 nl.talsmasoftware.context.mdc
+nl.talsmasoftware.context.slf4j.mdc
```

Comment:
Is this actually needed?

Reply:
You mean the new package? I think it more intuitively corresponds to the new module name. Also, if we're adding module-info in the future, this will make it easier to comply to the rule that one package may live in only one module.

Reply:
No I mean whether this `package-list` file is actually still used and this entry is needed.
Sorry for being imprecise.

Reply:
Well, there are problems with the latest maven javadoc plugin (see #157) which need to be resolved.
By then, the package list my not be needed anymore. However until then with the current javadoc plugin it still is, unfortunately.